src/client/containers/SignUp/index.js

```diff
@@ -53,5 +56,22 @@ export default function SignUpContainer() {
     }
   };
   if (isLoading) return <Loader />;
-  return <SignUp onSubmit={onSubmit} />;
+  return (
+    <div>
+      <div className="split sign-up-left">
+        <div>
+          <SideBanner
+            image={signUpImg}
+            style={{ backgroundColor: '#F1C644' }}
+            className="side-banner-signUp"
```

Comment:
I couldont find the class "side-banner-signup" in signUp.css

Reply:
oh good catch, I will remove it